router/gossip.go

```diff
@@ -0,0 +1,105 @@
+package router
+
+import (
+	"log"
+)
+
+type Gossip interface {
+	// specific message from one peer to another
+	// intermediate peers should relay it using unicast topology.
+	GossipUnicast(dstPeerName PeerName, buf []byte) error
+	// intended for a state change that everyone should hear about quickly
```

Comment:
There's an inconsistency here with the above description of unicast; there we talk about what it is, here about the purpose.